ui-kit/GroupCard/GroupCard.styles.js

```diff
@@ -57,6 +60,7 @@ const GroupCardTitle = styled.div`
 Card.AvatarCount = AvatarCount
 Card.DateTimeLabel = DateTimeLabel
 Card.GradientBackground = GradientBackground
+Card.GroupCard = GroupCard
 Card.GroupCardContent = GroupCardContent
 Card.GroupCardTitle = GroupCardTitle
```

Comment:
I didn't notice this before, but these should be `GroupCard` so then `Styled` is set to `GroupCard` at the top, which means you don't have to do `Styled.GroupCard` then.